src/components/header/Header.js

```diff
@@ -103,10 +103,18 @@ class Header extends Component {
     this.setState({ isInboxOpen: !!state });
   };
 
+  handleInboxToggle = () => {
+    this.setState({ isInboxOpen: !this.state.isInboxOpen });
+  };
+
   handleUDOpen = state => {
     this.setState({ isUDOpen: !!state });
   };
 
+  handleUDToggle = () => {
```

Comment:
Can you add a comment what UD stands for ? I assume it's User's Dropdown ?

Reply:
I have no idea, just went with the existing naming in the code. That illustrates your point well, though 😄